examples/np/cem_cartpole.py

```diff
@@ -16,9 +16,9 @@
 from garage.tf.samplers import OnPolicyVectorizedSampler
 
 
-def run_task(*_):
+def run_task(snapshot_config, *_):
```

Comment:
for running a task, the LocalRunner is inside your experiment method. for resuming a task, the LocalRunner is outside your experiment method. how do you plan on resolving this API in the future?

Reply:
I plan to move the LocalRunner for resume inside run_experiment(). I already finished it, but I'd better bring up after this.

Reply:
ok i think this is outside the scope of this PR, but we definitely need to sit down and plan this API for the future.

Reply:
okay great -- then ignore my comments about the API for now.